src/plugins/front-matter/index.js

```diff
@@ -1,6 +1,14 @@
 import parser from './parser';
 
 const install = function(hook, vm) {
+  // Used to remove front matter from embedded pages if installed.
+  vm.config.frontMatter = {};
+  vm.config.frontMatter.installed = true;
+  vm.config.frontMatter.parseMarkdown = function(content) {
+    const { body } = parser(content);
+    return body;
+  };
+
```

Comment:
it need consider that whether user can config the `config.frontMatter.installed`.

Reply:
I thought about that but there is really no benefit to the user changing this value. I hunted for a internal property that listed the installed plugins but found nothing. If there was a internal registry of the plugins that were added I would have checked that. 

Logic is that if you have front-matter installed you do not want the YAML headers returned on the page so therefore you would not want them in an include. If you do not have front-matter installed then you will see them on normal pages as well as includes.

Reply:
yep, ur right.